### DISCUSSION TITLE
Adding O-Counter button to the slideshow view for images.

### DISCUSSION THREAD

**Discussion by Hill-Brandon-M** [stashapp/stash#2240] [Ideas]
I wanted to make the changes myself, but it's going to take me a while since it's my first time using React... Does anyone else think this would be a good feature?

**Comment by WithoutPants** [comment]
I think the lightbox/slideshow feature should optionally show image metadata and allow setting rating, in addition to the o-counter. The lightbox feature as is is fairly basic, so it'd be good to develop it further. Something I would like to do when I have the time.

**Reply by sleetx** [reply]
This is a good idea. Right now in the slideshow view, it's not possible to open the image's page to view the metadata. Maybe just add a button on the slideshow view that links to the image's page where all that stuff is accessible.

**Comment by philpw99** [comment]
OMG !! What a hidden gem !! I cannot believe what I see after hitting "Wall" mode. It's truly AWESOME !!
This feature should be in the most prominent places. Or even set a big buttons says **"View Slide Show"**
@WithoutPants  you are too modest !
By the way, if such a big button exists, once you click it, please make the slide show full screen by default.
